test/server-unit/db.spec.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../../server/lib/db');
+const { expect } = require('chai');
+var _ = require('lodash');
+
+var dbTest = function () {
+
+  before(function () {
+    var params = {
```

Comment:
Ideally, we should use the values found in `.env.development`.  You can do that like this:
```js
require('dotenv').config({ path : '../../.env.development' });

// process.env.DB_NAME and other environmental variables are now defined. 
// We can just load db with a require() and everything will work as expected.

const db = require('../../server/lib/db');
```